### PR TITLE
[KRAKEN]: Structure for DB tests, PG db launch

### PR DESCRIPTION
**Pull request by kratos7**
Signed-off-by: Abdullah Shah shah.abdullah.syed@gmail.com

### CODE REVIEW

File: src/main/python/hydra/kraken/plumgrid/pg_kraken_tests.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        for auto-deletion, pre check for existing apps etc
+        """
+        self.broker_name_list = []
+        self.broker_nodes = 3
+        for x in range(self.broker_nodes):
+            app_name = "broker" + "-%d" % x
+            self.h_han.add_appid(app_name)
+            self.broker_name_list.append(app_name)
+
+    def check_pg_dht_node_health(self, ip):
```
Comment (sushilks) [inline src/main/python/hydra/kraken/plumgrid/pg_kraken_tests.py]: change to check_pg_broker_node_health.

File: src/main/python/hydra/kraken/plumgrid/pg_kraken_tests.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        ws_port = "12359"
+        ws_ip_port = ip + ":%s" % ws_port
+        ws = websocket.create_connection("ws://" + ws_ip_port)
+        ws.send("/health")
+        rep = ws.recv()
+        ws.close()
+        rep = json.loads(rep)
+        return rep["service_health"]
+
+    def assert_pg_dht_green(self, ip_list, timeout=20):
```
Comment (sushilks) [inline src/main/python/hydra/kraken/plumgrid/pg_kraken_tests.py]: change to assert_pg_broker_green

File: src/main/python/hydra/kraken/plumgrid/pg_kraken_tests.py (hunk 1 above)
Comment (kratos7) [inline src/main/python/hydra/kraken/plumgrid/pg_kraken_tests.py]: done

File: src/main/python/hydra/kraken/plumgrid/pg_kraken_tests.py (hunk 2 above)
Comment (kratos7) [inline src/main/python/hydra/kraken/plumgrid/pg_kraken_tests.py]: done

**Pull request closed without merging**